### PR TITLE
[IOTDB-2842] SHOW PIPE PipeName throws NPE if receiverService isn't initialize

### PR DESCRIPTION
**Pull request by Cpaulyz**

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/sync/receiver/manager/ReceiverManager.java [hunk 1]
```diff
@@ -93,7 +93,13 @@ public void dropPipe(String pipeName, String remoteIp) throws IOException {
   }
 
   public List<PipeInfo> getPipeInfos(String pipeName) {
-    return new ArrayList<>(pipeInfoMap.get(pipeName).values());
+    List<PipeInfo> res;
+    if (pipeInfoMap.containsKey(pipeName)) {
+      res = new ArrayList<>(pipeInfoMap.get(pipeName).values());
+    } else {
+      res = new ArrayList<>();
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/sync/receiver/manager/ReceiverManager.java:100]:
```suggestion
      res = Collections.emptyList();
```

Comment (Cpaulyz) [reply server/src/main/java/org/apache/iotdb/db/sync/receiver/manager/ReceiverManager.java]: done

File: server/src/main/java/org/apache/iotdb/db/sync/receiver/manager/ReceiverManager.java [hunk 2]
```diff
@@ -28,10 +28,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/sync/receiver/manager/ReceiverManager.java:31]: We don't suggest to use * ...

Comment (Cpaulyz) [reply server/src/main/java/org/apache/iotdb/db/sync/receiver/manager/ReceiverManager.java:31]: done

**Review by HTHou — APPROVED** [review]

**Pull request merged**